### DISCUSSION TITLE
Best practices for sending Delayed Messages？

### DISCUSSION THREAD

**Discussion by l-7-l** [apache/iggy#1077] [Q&A] [answered]
Is the support for delayed messages planned for the near future? Are there any good workarounds before that?

**Comment by spetz** [comment, marked as answer]
Hey,
We don't have such a feature, since the message stream is just an append-only log, not a sophisticated broker message which does server a different purpose. It could be done only on the client-side application, by sending the delayed messages e.g. via some cron job or so.
